pkg/oc/oc.go

```diff
@@ -67,6 +73,45 @@ func UpdateTektonConfigwithInvalidData(patch_data, errorMessage string) {
 	}
 }
 
+func AssertCronjobPresent(c *clients.Clients, cronJobName, namespace string) {
```

Comment:
Does it make sense to also check that there is only one cron job with given name?

Reply:
I thought about it, but the problem is if we have annotated any namespace with the pruner annotations and if there is any failure in deleting the namespace, there will be more than one cronjobs